code/game/objects/buckling.dm

```diff
@@ -92,8 +104,11 @@
 		return 1
 
 
-/atom/movable/proc/user_unbuckle_mob(mob/user)
-	var/mob/living/M = unbuckle_mob()
+//Edit so I can see all the calls
+/atom/movable/proc/user_unbuckle_mob(mob/living/buckled_mob, mob/user)
+	if(!buckled_mob && buckled_mobs.len)
+		buckled_mob = buckled_mobs[1]
```

Comment:
Doesn't this mean that if I click cancel when shown the window to chose which of the mob to unbuckle it would still unbuckle someone?